src/App.css

```diff
@@ -130,3 +128,54 @@ button {
   gap: 1px;
   background-color: #c9c9c9;
 }
+
+/* START Home */
+
+.Home {
+  padding-top: 10%;
+}
+
+/* END Home */
+
+/* START Navigation bar */
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  background-color: rgb(255, 255, 253);
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+}
+
+li {
+  margin: 0 5px;
+}
+
+.mm-logo {
+  width: 80px;
+}
```

Comment:
I cannot give inline comments on this PR that's why I am using the random file. 
- [ ] Let's make our Readme descriptive. Currently, you have autogenerated Readme.